scripts/check_lint.sh

```diff
@@ -48,18 +48,30 @@ trap clean EXIT
 for i in $(find $lint_path $exclude -name "go.mod")
 do
   pushd "$(dirname $i)" > /dev/null
-  if [ -z $(go list) ]; then
+  if [ -z $(go list 2> /dev/null) ]; then
+      echo "skipped $(dirname $i)"
       popd  > /dev/null
       continue
   fi
 
-  ${DOCKER_CMD} run --rm -e GOPROXY=${GOPROXY} \
-  -v "$ROOT:/opt/workspace" \
-  -v "$ROOT/.golangci.yml:/opt/workspace/$(dirname $i)/.golangci.yml" \
-  -w "/opt/workspace/$(dirname $i)" \
-  ${lint_image} golangci-lint run
+  if [ -f "$ROOT/$(dirname $i)/.custom_golangci.yml" ]; then
```

Comment:
Automation improvement - a subfolder that's tested separately (ie, it's a separate module, so the `find` above will list it) can contain a `.custom_golangci.yml` file, for custom golangci configuration. This is necessary here as `canislabs/golangci-lint:latest`, used for linting the indy vdr, is not up to date with `golangci/golangci-lint:v1.39.0`, and thus cannot use the same config yaml due to not recognizing some of the linters listed in the main `.golangci.yml` file.